src/bootstrap/native.rs

```diff
@@ -261,7 +261,7 @@ impl Step for Llvm {
         let llvm_targets = match &builder.config.llvm_targets {
             Some(s) => s,
             None => {
-                "AArch64;ARM;BPF;Hexagon;MSP430;Mips;NVPTX;PowerPC;RISCV;\
+                "AArch64;ARM;BPF;Hexagon;LoongArch;MSP430;Mips;NVPTX;PowerPC;RISCV;\
```

Comment:
Move LoongArch into `llvm_exp_targets` below.

Reply:
The llvm used by the build does not contain any support for loongarch64, adding llvm target support for loongarch64 in config.toml.example and src/bootstrap/native.rs will cause the build to fail, and these two files are eventually removed.
I will submit a separate PR and fix these issues